test/testenv/verificationutils.go

```diff
@@ -441,6 +441,36 @@ func VerifySecretsUpdatedOnPod(deployment *Deployment, testenvInstance *TestEnv,
 	}
 }
 
+// VerifyNewSecretValueOnVersionedSecretObject Check whether the new versioned secret object is created with new value
```

Comment:
There is no verification for versioned secret in this method ?

Reply:
Its indirect verification as we check for values on the new versioned object. If the object is not created the call would return an empty string.
The function as the name states checks if the new secret value is present on Versioned secret object

Reply:
Since there is no check for new versioned secret I think the description is inaccurate. We should describe the actual logic inside the method.

Reply:
Also if we decide to create a method for versioned secret that should be independent of data(secret key/value) verification which can distinguished through a flag.

Reply:
modified the method name. We can add a method to check for the versioned secret object in another PR